net.carrotindustries.usbkvm.json

```diff
@@ -0,0 +1,145 @@
+{
+    "app-id": "net.carrotindustries.usbkvm",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "46",
```

Comment:
current is 47.

Reply:
In 47, v4l devices aren't found by gstreamer for some reason and video capture is an essential part of this application. To reproduce this, run `flatpak run --device=all  runtime/org.gnome.Platform/x86_64/47` . In the flatpak shell, run `gst-device-monitor-1.0`. 

This may be a problem of the underlying freedesktop runtime though: In 23.08, v4l works in gstreamer. In 24.08 it does not.

Reply:
I ran that and got a bunch of "device founds", so seems to be working for me?

Reply:
Ah I guess you meant v4l2 specifically, can you open an issue in https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/issues ?

Reply:
Ok, opened an issue: https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/issues/1824